contrib/migration/README.md

```diff
@@ -0,0 +1,55 @@
+# BM HCP Migration Script (Alpha)
+
+## Disclaimer
+
+This script is a targeted solution meant as a stop-gap and designed to enable functionality for specific cases as an interim measure.
+
+Full support will ultimately be provided through OADP as part of the long-term solution.
+
+## Prerequisites
+
+- >= MCE 2.8 (For now)
```

Comment:
4.18? I thought it works for 4.17 (to be released Nov 5th 2024?)

Reply:
Not 100%, @CrystalChun needs to backport something to Assisted Installer to make possible the node adoption in the new management cluster.

Reply:
Once this PR https://github.com/openshift/assisted-service/pull/6968 merges, it'll be available in MCE 2.7

Reply:
Ok, let me change the versions then. Thanks folks :)